tests/main/test_views.py

```diff
@@ -6,3 +6,10 @@ def test_index(client, admin_client):
     with assertTemplateUsed(template_name="main/index.html"):
         response = client.get("/")
     assert response.status_code == 200
+
+
+def test_privacy(client, admin_client):
```

Comment:
I think we should set up some more Django-friendly testing boilerplate to make these tests less repetitive in the future.

Reply:
Opened a separate issue for this